frontend/test/ui/conftest.py

```diff
@@ -17,11 +21,35 @@ def firefox_options(firefox_options):
         'extensions.install.requireBuiltInCerts', False)
     firefox_options.set_preference('xpinstall.signatures.required', False)
     firefox_options.set_preference('extensions.webapi.testing', True)
-    firefox_options.set_preference('example.com', 'local')
+    firefox_options.set_preference('testpilot.env', 'local')
+    firefox_options.add_argument('-foreground')
     firefox_options.log.level = 'trace'
     return firefox_options
 
 
+@pytest.yield_fixture(scope='session', autouse=True)
+def enable_dev_experiment():
```

Comment:
This isn't called anywhere. I can kind of guess that the decorator automatically applies it, but can you clarify?

scope='session' means that it gets run once per session, and autouse means it gets used?

Does the code before the yield run at the beginning of the session, and the code after the yield run at the end of the session?